src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -21,8 +21,13 @@ namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
     public class DocumentDBConfiguration : IExtensionConfigProvider
     {
         internal const string AzureWebJobsDocumentDBConnectionStringName = "AzureWebJobsDocumentDBConnectionString";
+        // We separate the Trigger connection string in case of a mix of Trigger & output Attribute, otherwise it would generate an infinite loop
```

Comment:
If you set the DocumentDBTrigger to have the same connection string as the DocumentDB binding, it creates an infinite loop. You write a document, the trigger fires, you work with the document that changed and use the output binding so save, since both point to the same service, it might trigger again (if both work on the same collection). That happens if I reuse the same setting as the output binding on the trigger.